pkg/pillar/cmd/zedrouter/zedrouter.go

```diff
@@ -2616,3 +2663,70 @@ func handleDNSDelete(ctxArg interface{}, key string,
 	maybeHandleDNS(ctx)
 	log.Infof("handleDNSDelete done for %s\n", key)
 }
+
+func validateAppNetworkConfig(ctx *zedrouterContext, appNetConfig types.AppNetworkConfig,
```

Comment:
Makes sense to have it return true if the config is valid.

Reply:
sure